shared/common-adapters/markdown/index.native.js

```diff
@@ -188,7 +188,11 @@ class OriginalMarkdown extends React.PureComponent<Props> {
         return createComponent('', '', content, {}) || null
       } else {
         return (
-          <Text type="Body" style={this.props.style} allowFontScaling={this.props.allowFontScaling}>
+          <Text
+            type="Body"
+            style={collapseStyles([styles.neutral, this.props.style])}
```

Comment:
I'm a little confused why we have `styles.neutral` in here at all? What is it doing if it is just empty? Does it get set somewhere else?

Reply:
it effectively does nothing but there's no point in cleaning this up. next release will have simple-markdown